apps/ledger-live-mobile/src/components/ReceiveAccountCard.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import {
+  getAccountName,
+  getAccountSpendableBalance,
+} from "@ledgerhq/live-common/lib/account";
+import {
+  getAccountCurrency,
+  getAccountUnit,
+} from "@ledgerhq/live-common/lib/account/helpers";
+import { getTagDerivationMode } from "@ledgerhq/live-common/lib/derivation";
+import { Account, CryptoCurrency } from "@ledgerhq/live-common/lib/types";
+import { Flex, Text } from "@ledgerhq/native-ui";
+import { useTheme } from "styled-components/native";
+import { TouchableOpacity } from "react-native-gesture-handler";
+
+import Card, { Props as CardProps } from "./Card";
+import CurrencyIcon from "./CurrencyIcon";
+import CurrencyUnitValue from "./CurrencyUnitValue";
+
+const Tag = ({ children, ...props }: { children: string }) => (
+  <Flex p={2} bg={"neutral.c30"} borderRadius={2} {...props}>
+    <Text variant="tiny" fontWeight="medium" color="neutral.c80" uppercase>
+      {children}
+    </Text>
+  </Flex>
+);
+
+export type Props = CardProps & {
+  account: Account;
+  style?: any;
+  disabled?: boolean;
+  useFullBalance?: boolean;
+  AccountSubTitle?: React.ReactNode;
+};
+
+const AccountCard = ({
+  onPress,
+  account,
+  style,
+  disabled,
+  useFullBalance,
+  AccountSubTitle,
+  ...props
+}: Props) => {
+  const { colors } = useTheme();
+  const currency = getAccountCurrency(account);
+  const unit = getAccountUnit(account);
+  const tag =
+    account.derivationMode !== undefined &&
+    account.derivationMode !== null &&
```

Comment:
to me this is not possible. derivationMode is always set and is a string. Do you know why you had to do this?